### ISSUE TITLE
Internal RuboCop warning for `AbcSize` in `Migration/DepartmentName` examples

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#14931]
Running `bundle exec rake default` triggers a warning in `Migration/DepartmentName` cop regarding a missing department for `AbcSize`.

This issue seems to have been introduced in commit [`97ed44a557c6b86a05c01283feb84103ff687f41`](https://github.com/rubocop/rubocop/commit/97ed44a557c6b86a05c01283feb84103ff687f41).
Previously, this warning was not present when running `rake default`.

--------

## Steps to reproduce the problem

Run `bundle exec rake default`.

## Expected behavior

No warnings.

## Actual behavior

The following warning is displayed:

> lib/rubocop/cop/migration/department_name.rb: Warning: no department given for AbcSize. Run rubocop -a --only Migration/DepartmentName to fix.

## RuboCop version

```
$ [bundle exec] rubocop -V
1.84.2 (using Parser 3.3.10.0, rubocop-ast 1.49.0, analyzing as Ruby 2.7, running on ruby 3.4.8) [x86_64-linux]
  - rubocop-performance 1.26.1
  - rubocop-rake 0.7.1
  - rubocop-rspec 3.9.0
```
